### PR TITLE
Fix pgvector function example code

### PR DESCRIPTION
**Pull request by nathanrchn**
Fix the function argument names in the pgvector example from `similarity_threshold` to `match_threshold` to be consistent with the typescript example below.

Remove the ambiguity with the `id` and `content` row names by adding the prefix `documents.`

Change the return order by adding the variable `similarity` in place of the expression and thus adding the return in descending order

### CODE REVIEW

File: apps/www/_blog/2023-02-03-openai-embeddings-postgres-vector.mdx [hunk 1]
```diff
@@ -117,12 +117,12 @@ as $$
 begin
   return query
   select
-    id,
-    content,
+    documents.id,
+    documents.content,
     1 - (documents.embedding <=> query_embedding) as similarity
   from documents
   where 1 - (documents.embedding <=> query_embedding) > similarity_threshold
```
Comment (kiwicopple) [inline apps/www/_blog/2023-02-03-openai-embeddings-postgres-vector.mdx:124]: does this also need to change to `match_threshold`?

Comment (nathanrchn) [reply apps/www/_blog/2023-02-03-openai-embeddings-postgres-vector.mdx:124]: Yes the variable name `similarity_threshold` should also be changed to `match_threshold` in the `where` clause to ensure consistency with the examples later in the post.

Comment (kiwicopple) [reply apps/www/_blog/2023-02-03-openai-embeddings-postgres-vector.mdx:124]: would you be able to make the change @nathanrchn ? then I can merge it in! thanks 🙏

Comment (nathanrchn) [reply apps/www/_blog/2023-02-03-openai-embeddings-postgres-vector.mdx]: Of course I didn't understand your comment at first, sorry for that.

**Pull request closed without merging**